system/Database/Forge.php

```diff
@@ -990,7 +995,7 @@ protected function _attributeAutoIncrement(array &$attributes, array &$field)
         }
     }
 
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table. This is used throughout this PR.